sdk/resourcemanager/Azure.ResourceManager.Core/src/Resources/ResourceIdentity.cs

```diff
@@ -76,136 +70,6 @@ public ResourceIdentity(SystemAssignedIdentity systemAssigned, IDictionary<Resou
         /// </summary>
         public IDictionary<ResourceGroupResourceIdentifier, UserAssignedIdentity> UserAssignedIdentities { get; private set; }
 
-        /// <summary>
```

Comment:
I think the ctor on 49 should be internal and the serialization ctor, 30 should be initialization, and 19 shouldn't exist?

Reply:
Discussed offline and updated as per the discussion.